lua/nvim-autopairs/completion/cmp.lua

```diff
@@ -8,16 +8,20 @@ M.setup = function(opt)
         map_cr = true,
         map_complete = true,
         auto_select = true,
+        insert = false,
         map_char = { all = '(', tex = '' },
     }, opt)
     local map_cr = opt.map_cr
     local map_complete = opt.map_complete
     local map_char = opt.map_char
+    local behavior = opt.insert
+        and cmp.ConfirmBehavior.Insert
```

Comment:
hmn i think it should be replace.

Reply:
I think it's confusing here.

Maybe something like this could be clearer : 

```lua
require("nvim-autopairs.completion.cmp").setup{
  map_cr = true,
  map_complete = true,
  auto_select = true,
  confirm_behavior = cmp.ConfirmBehavior.Insert,
}
```

What do you think ?